lib/hound/request_utils.ex

```diff
@@ -1,7 +1,9 @@
 defmodule Hound.RequestUtils do
   @moduledoc false
 
-  def make_req(type, path, params \\ %{}, options \\ %{}, retries \\ 0)
+  @retries Application.get_env(:hound, :retries, 0)
```

Comment:
Could we make `retries` a function rather than a module attribute?
IIRC when it's a module attribute and it's changed is the config, a clean build is needed for the changes to be picked up.

Reply:
Yes, you are right! I'll change it right away